bin/import_cv2.py

```diff
@@ -31,11 +31,14 @@
 
 
 def _preprocess_data(tsv_dir, audio_dir, space_after_every_character=False):
-    for dataset in ["train", "test", "dev", "validated", "other"]:
-        input_tsv = os.path.join(os.path.abspath(tsv_dir), dataset + ".tsv")
-        if os.path.isfile(input_tsv):
-            print("Loading TSV file: ", input_tsv)
-            _maybe_convert_set(input_tsv, audio_dir, space_after_every_character)
+    exclude = []
+    for dataset in ["test", "dev", "train", "validated", "other"]:
+        set_samples = _maybe_convert_set(dataset, tsv_dir, audio_dir, space_after_every_character)
+        if dataset in ["test", "dev"]:
+            exclude += set_samples
+        if dataset == "validated":
+            _maybe_convert_set("train-all", tsv_dir, audio_dir, space_after_every_character,
+                               rows=set_samples, exclude=exclude)
```

Comment:
Would be nice to split the read+convert part and the write CSV part into two functions so that `_maybe_convert_set` does not have to act like two different functions depending on the `rows` parameter. But since this is an importer I don't particularly care.